### ISSUE TITLE
Deaccession fields

### ISSUE THREAD

**Issue by acvaughan** [hiscom/hispid#15]
We need to include a deaccession flag field in HISPID to flag when records need to be flagged as deaccessioned in AVH. For the remaining occurrence record to be usefully interpreted by users, the reason for deaccessioning the specimen should be recorded in a 'deaccession reason' field. (From the minutes of the 2012 HISCOM meeting: http://hiscom.rbg.vic.gov.au/wiki/HISCOM_2012_AGM_Canberra_minutes#11._HISPID)

**Comment by acvaughan** [comment]
I think differentiating between Deaccessioned specimens and Erroneous records will cover the Deaccession reason: from my understanding, it was about being able to tell the difference between things that were just wrong, or that were okay but have been destroyed somehow.

**Comment by nielsklazenga** [comment]
You are not wrong, but the premise for this was that both deaccessioned records and erroneous records are covered by the standard. That premise is wrong. Erroneous records are deleted from the database, so there is no record to exchange and there is no point in having a field but no record to put it in. Any concept in HISPID can only cover (in this case) deaccessioned specimens for which there is still a record that can be exchanged. Deleting records that have been delivered erroneously has to go outside the normal data exchange process and the only information that is needed is the identifier for the record to be deleted (in AVH that is institutionCode and catalogNumber).

I am very well aware that at the 2012 meeting I was the one who thought it would be easiest to add the deleted records to the exchange data (not HISPID though). However, ALA really doesn't want to deal with it that way – so there goes easy – and I think now they are right.

**Comment by nielsklazenga** [comment]
Hobart, 2015-10-20:
Comment: "Can be used to indicate that a specimen is missing or deaccessioned."
Vocabulary: "missing", "deaccessioned".

**Comment by nielsklazenga** [comment]
HISCOM 2016: maps to dwc:disposition
